### PR TITLE
Update global.css to add PCB background

### PR DESCRIPTION
**Pull request by jzaleta**
Fixes the errors caused on #1031 and properly adds the PCB background!

### CODE REVIEW

**Comment by jzaleta** [comment]
Okay I think this PR is ready! @LucasHT22 @kognise 🎉

**Comment by jzaleta** [comment]
I managed to fix the pcb background to make it less dark without directly making changes to the file, I did this to avoid having issues with the file itself, because some hack clubbers use the pcb file as their Scrapbook background image.

**Comment by jzaleta** [comment]
Captura de pantalla 2023-06-02 a la(s) 13 30 43

Captura de pantalla 2023-06-02 a la(s) 13 31 17

**Comment by jzaleta** [comment]
Hi @kognise, could you please check this PR when you have some time?
Thanks!

File: src/global.css [hunk 1]
```diff
@@ -29,10 +29,24 @@ body { overflow: auto; }
 	--line-height-copy: 1.5;
 }
 
+html::before {
```
[the remark this answers is not in the document]
Comment (jzaleta) [reply src/global.css:32]: Yeah that wasn't intended. I think this won't work well sadly :(

[the remark this answers is not in the document]
Comment (grymmy) [reply src/global.css:32]: @jzaleta Perhaps if you limit the scope to only the specific divs you want to have modified, this would work?

**Pull request closed without merging**